crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -370,6 +370,10 @@ impl<I: SparseSetIndex, V> SparseSet<I, V> {
     pub fn values_mut(&mut self) -> impl Iterator<Item = &mut V> {
         self.dense.iter_mut()
     }
+
+    pub fn sparse_array(&self) -> &SparseArray<I, usize> {
```

Comment:
Ditto here.

Reply:
I disagree I would expect to be able to access this part of a sparse set, consider this PR which needed to access it :P realistically sparse set stuff _could_ be in a different crate.

Reply:
Given that we don't currently care about semver (and hopefully never do), I'm fine with that stance.

Reply:
Again, as stated elsewhere. The internal indices really don't need exposing here. We already have another way of checking if an index is present in the SparseSet. No reason to add another, potentially more confusing, API to do the exact same thing.

Reply:
I agree that the `usize` bit isnt important but the sparseset impl isnt setup to give us a `Bitset<I>` so this is the best we can do I think

Reply:
Which is why I'm suggesting we make this `pub(crate)`, because the only real use for this conflicts with `SparseSet::contains`.